tools/make-debs

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+# This file is part of Cockpit.
+#
+# Copyright (C) 2021 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+set -euf
+
+mydir=$(dirname $(realpath "$0"))
+
+usage()
+{
+	echo "usage: make-debs [--quick] [tarball]"
+}
+
+args=$(getopt -o "h,q" -l "help,quick" -- "$@")
+eval set -- "$args"
+while [ $# -gt 0 ]; do
+	case $1 in
+	-q|--quick)
+		export DEB_BUILD_OPTIONS=nocheck
+		;;
+	-h|--help)
+		usage
+		exit 0
+		;;
+	--)
+		shift
+		break
+		;;
+	esac
+	shift
+done
+
+if [ $# -gt 1 ]; then
+	usage
+	exit 2
+fi
+
+if [ -z "${1:-}" ]; then
+    source=$("$mydir/../test/make_dist.py" | tail -n1)
+else
+    source="$(realpath "$1")"
+fi
+
+version=$(echo "$source" | sed -n 's|.*cockpit-\([^ /-]\+\)\.tar\..*|\1|p')
+if [ -z "$version" ]; then
+    echo "make-debs: couldn't parse version from tarball: $source"
+    exit 2
+fi
+
+builddir=tmp/builddeb
+rm -rf "$builddir"
+mkdir -p "$builddir"
+olddir=$(pwd)
+cd "$builddir"
+tar --strip-components=1 -xf "$source"
+cp -r tools/debian .
+# plug in our version
+sed -i "1 s/(0-/(${version}-/" debian/changelog
+dpkg-buildpackage -us -uc -b -j$(nproc)
+cd "$olddir"
```

Comment:
`cd -` also works here, just to avoid storing olddir.

Reply:
Done, thanks!